src/actions/player/playerTrendsActions.js

```diff
@@ -63,9 +68,21 @@ export const getPlayerTrends = (playerId, options = {}, fieldName) => (dispatch)
           start_time: match.start_time,
           win,
         });
-
         return dataList;
       }, []);
+
+      // Compute in reverse order so that first n can be discarded
+      for (let i = trends.length - 1; i > chunkSize - 1; i -= 1) {
+        trends[i].value = (trends[i].value - trends[i - chunkSize].value) / chunkSize;
+
+        // Update graph index so it starts at 1 (since we only display 480 at a time)
+        trends[i].x -= chunkSize;
+      }
+
+      // Discard first 20 elements
+      trends.splice(0, chunkSize);
```

Comment:
nvm, I was looking at the return value of the function, not the mutated value of trends.